test/test_unidler.py

```diff
@@ -0,0 +1,253 @@
+from io import BytesIO
+from unittest.mock import MagicMock, call, patch
+
+import pytest
+
+import unidler
```

Comment:
Do you need this `import unidler` rule if in the following line it's importing from the same module but specifying what to import?

Reply:
It makes it easier to patch the functions in the idler module. I'm importing the whole `unidler` module and calling the functions in the tests like `unidler.unidle_deployment`.
I import the constants directly because it's shorter to say `labels[IDLED]` than `labels[unidler.IDLED]` and I don't need to patch them.
It's a little unusual to import the whole module and some of its contents, I suppose.